plugins/sigma.exporters.xlsx/sigma.exporters.xlsx.js

```diff
@@ -220,6 +217,9 @@
    * @return {string}        The Spreadsheet string.
    */
   sigma.prototype.toXLSX = function(params) {
+      if (typeof XLSX === 'undefined')
```

Comment:
I'd prefer to throw error early if the file is included by the dependency is missing.